### DISCUSSION TITLE
Can not integrate iota and rewards from old wallet into new firefly wallet

### DISCUSSION THREAD

**Discussion by m33394jut** [iotaledger/firefly#3094] [Help & Support]
Hi, i setup the new firefly wallet with profile but only can create new phassphrase, also inside there is no possibility to integrate the coins and rewards from my wallet before !!!! How can i restore my old account on the new firefly wallet with my iotas and rewards?

**Comment by MiKeF05** [comment]
With integrate coins you mean migrate them from your old Trinity wallet? When you create a new profile you need to choose "Migrate or restore a wallet". 
![image](https://user-images.githubusercontent.com/71659397/173658527-ddce117d-cb4c-409a-a993-3970bb8c5dc6.png)


If you need more info how to migrate your tokens here is a step-by-step guide:
https://blog.iota.org/firefly-token-migration/ (81 char seed)
https://blog.iota.org/firefly-token-migration-guide-for-ledger-users/ (Ledger device)
